awsemo/spiders/awsemo_spider.py

```diff
@@ -6,5 +6,5 @@ class AwsemoSpider(scrapy.Spider):
     start_urls = ['https://github.com/sindresorhus/awesome']
 
     def parse(self, response):
-        for url in response.css('#readme a::attr(href)').extract():
-            self.log(url + '/n')
+        for u in response.css('#readme a::attr(href)').extract():
```

Comment:
I doubt it didn't work because of the variable name. Let's verify it again because `u` isn't a very verbose variable name.